### PR TITLE
Pass global template variables via context processor.

### PR DESCRIPTION
**Pull request by jmcarp**
Avoid passing template globals to each view manually, which turns out to
be easy to forget.

TODO: restore tests. Pick better namespace?

### CODE REVIEW

**Comment by jmcarp** [comment]
Tests restored. What do you think about exposing these variables as `globals.ANALYTICS`, `globals.APP_PREFIX`, etc. @cmc333333 ?

File: regulations/context.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from six.moves.urllib_parse import urlencode
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+
+def globals(request):
+    env = 'source' if getattr(settings, 'JS_DEBUG', False) else 'built'
+    prefix = reverse('regulation_landing_view', kwargs={'label_id': '9999'})
+    prefix = prefix.replace('9999', '')
```
Comment (cmc333333) [inline regulations/context.py]: Super ugly. I bet there's a better way to determine this.

**Comment by cmc333333** [comment]
Ideally we'd have a namespace specific to eregs stuff, e.g. `EREGS_GLOBALS` or similar -- the original issue was that we weren't running on our own Django, we were just an app. That said, I'm happier to get this in as is.

**Comment by jmcarp** [comment]
Should I rename to `EREGS_GLOBALS`? Just a quick find and replace.

**Comment by cmc333333** [comment]
:+1: Also, related -- then you're not redefining a top-level Python function :D

**Pull request merged**